Deprecated lambda capture (as of c++20) in use

```
[ 57%] Building CXX object CMakeFiles/Attorney_Online.dir/src/charselect.cpp.o
/home/marisa/AO2-Client/src/aoimage.cpp: In lambda function:
/home/marisa/AO2-Client/src/aoimage.cpp:15:43: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   15 |     connect(movie, &QMovie::frameChanged, [=]{
      |                                           ^
/home/marisa/AO2-Client/src/aoimage.cpp:15:43: note: add explicit ‘this’ or ‘*this’ capture
[ 59%] Building CXX object CMakeFiles/Attorney_Online.dir/src/chatlogpiece.cpp.o
[ 61%] Building CXX object CMakeFiles/Attorney_Online.dir/src/courtroom.cpp.o
[ 63%] Building CXX object CMakeFiles/Attorney_Online.dir/src/debug_functions.cpp.o
[ 65%] Building CXX object CMakeFiles/Attorney_Online.dir/src/demoserver.cpp.o
/home/marisa/AO2-Client/src/aooptionsdialog.cpp: In lambda function:
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:997:57: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
  997 |   connect(ui_mount_remove, &QPushButton::clicked, this, [=] {
      |                                                         ^
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:997:57: note: add explicit ‘this’ or ‘*this’ capture
/home/marisa/AO2-Client/src/aooptionsdialog.cpp: In lambda function:
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:1015:53: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
 1015 |   connect(ui_mount_up, &QPushButton::clicked, this, [=] {
      |                                                     ^
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:1015:53: note: add explicit ‘this’ or ‘*this’ capture
/home/marisa/AO2-Client/src/aooptionsdialog.cpp: In lambda function:
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:1033:55: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
 1033 |   connect(ui_mount_down, &QPushButton::clicked, this, [=] {
      |                                                       ^
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:1033:55: note: add explicit ‘this’ or ‘*this’ capture
/home/marisa/AO2-Client/src/aooptionsdialog.cpp: In lambda function:
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:1055:62: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
 1055 |   connect(ui_mount_clear_cache, &QPushButton::clicked, this, [=] {
      |                                                              ^
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:1055:62: note: add explicit ‘this’ or ‘*this’ capture
/home/marisa/AO2-Client/src/aooptionsdialog.cpp: In lambda function:
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:1060:68: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
 1060 |   connect(ui_mount_list, &QListWidget::itemSelectionChanged, this, [=] {
      |                                                                    ^
/home/marisa/AO2-Client/src/aooptionsdialog.cpp:1060:68: note: add explicit ‘this’ or ‘*this’ capture```

```
[ 36%] Building CXX object CMakeFiles/Attorney_Online.dir/src/aoemotebutton.cpp.o
/home/marisa/AO2-Client/src/aobutton.cpp: In lambda function:
/home/marisa/AO2-Client/src/aobutton.cpp:11:41: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   11 |   connect(movie, &QMovie::frameChanged, [=]{
      |                                         ^
/home/marisa/AO2-Client/src/aobutton.cpp:11:41: note: add explicit ‘this’ or ‘*this’ capture```

--- a/src/aobutton.cpp
+++ b/src/aobutton.cpp
@@ -8,7 +8,7 @@ AOButton::AOButton(QWidget *parent, AOApplication *p_ao_app)
 {
   ao_app = p_ao_app;
   movie = new QMovie(this);
-  connect(movie, &QMovie::frameChanged, [=]{
+  connect(movie, &QMovie::frameChanged, [this]{
     this->setIcon(movie->currentPixmap().scaled(this->size(), Qt::IgnoreAspectRatio, Qt::SmoothTransformation));
     this->setIconSize(QSize(this->width(), this->height()));
   });

--- a/src/aoimage.cpp
+++ b/src/aoimage.cpp
@@ -12,7 +12,7 @@ AOImage::AOImage(QWidget *parent, AOApplication *p_ao_app, bool make_static) : Q
   if (!is_static) // Only create the QMovie if we're non-static
   {
     movie = new QMovie(this);
-    connect(movie, &QMovie::frameChanged, [=]{
+    connect(movie, &QMovie::frameChanged, [this]{
       QPixmap f_pixmap = movie->currentPixmap();
       f_pixmap =
           f_pixmap.scaled(this->size(), Qt::IgnoreAspectRatio);

--- a/src/aooptionsdialog.cpp
+++ b/src/aooptionsdialog.cpp
@@ -854,7 +854,7 @@ AOOptionsDialog::AOOptionsDialog(QWidget *parent, AOApplication *p_ao_app)
   ui_mount_remove->setSizePolicy(stretch_btns);
   ui_mount_remove->setEnabled(false);
   ui_mount_buttons_layout->addWidget(ui_mount_remove, 0, 1, 1, 1);
-  connect(ui_mount_remove, &QPushButton::clicked, this, [=] {
+  connect(ui_mount_remove, &QPushButton::clicked, this, [this] {
     auto selected = ui_mount_list->selectedItems();
     if (selected.isEmpty())
       return;
@@ -872,7 +872,7 @@ AOOptionsDialog::AOOptionsDialog(QWidget *parent, AOApplication *p_ao_app)
   ui_mount_up->setMaximumWidth(40);
   ui_mount_up->setEnabled(false);
   ui_mount_buttons_layout->addWidget(ui_mount_up, 0, 3, 1, 1);
-  connect(ui_mount_up, &QPushButton::clicked, this, [=] {
+  connect(ui_mount_up, &QPushButton::clicked, this, [this] {
     auto selected = ui_mount_list->selectedItems();
     if (selected.isEmpty())
       return;
@@ -890,7 +890,7 @@ AOOptionsDialog::AOOptionsDialog(QWidget *parent, AOApplication *p_ao_app)
   ui_mount_down->setMaximumWidth(40);
   ui_mount_down->setEnabled(false);
   ui_mount_buttons_layout->addWidget(ui_mount_down, 0, 4, 1, 1);
-  connect(ui_mount_down, &QPushButton::clicked, this, [=] {
+  connect(ui_mount_down, &QPushButton::clicked, this, [this] {
     auto selected = ui_mount_list->selectedItems();
     if (selected.isEmpty())
       return;
@@ -912,12 +912,12 @@ AOOptionsDialog::AOOptionsDialog(QWidget *parent, AOApplication *p_ao_app)
   "Use this when you have added an asset that takes precedence over another "
   "existing asset."));
   ui_mount_buttons_layout->addWidget(ui_mount_clear_cache, 0, 6, 1, 1);
-  connect(ui_mount_clear_cache, &QPushButton::clicked, this, [=] {
+  connect(ui_mount_clear_cache, &QPushButton::clicked, this, [this] {
     asset_cache_dirty = true;
     ui_mount_clear_cache->setEnabled(false);
   });
 
-  connect(ui_mount_list, &QListWidget::itemSelectionChanged, this, [=] {
+  connect(ui_mount_list, &QListWidget::itemSelectionChanged, this, [this] {
     auto selected_items = ui_mount_list->selectedItems();
     bool row_selected = !ui_mount_list->selectedItems().isEmpty();
     ui_mount_remove->setEnabled(row_selected);

--- a/src/courtroom.cpp
+++ b/src/courtroom.cpp
@@ -1807,7 +1807,7 @@ void Courtroom::on_chat_return_pressed()
 
   ui_ic_chat_message->blockSignals(true);
   QTimer::singleShot(ao_app->get_chat_ratelimit(), this,
-                     [=] { ui_ic_chat_message->blockSignals(false); });
+                     [this] { ui_ic_chat_message->blockSignals(false); });
   // MS#
   // deskmod#
   // pre-emote#

